docs/src/other/glossary.md

```diff
@@ -135,22 +141,31 @@ and redeploying the parent.
 
 When you merge an environment, three things happen:
 
-* Any code changes are merged via Git to the parent branch.
-* Your apps rebuilt on the parent branch, if necessary.
+- Any code changes are merged via Git to the parent branch.
+- Your apps rebuilt on the parent branch, if necessary.
   (This is skipped if the same code with the same [variables](../development/variables/_index.md) has been built for any environment.)
-* The parent branch is deployed.
+- The parent branch is deployed.
 
 ## PaaS
 
 A Platform as a Service is an end-to-end hosting solution
 that includes workflow tools, APIs, and other functionality above and beyond basic hosting.
 The best example is Platform.sh (although we're a little biased).
 
+## Platform
+Platform or Platform.sh is the infrastructure which is running all your projects.
+
```

Comment:
:hammer_and_wrench: I wouldn't use `Platform` without the `.sh`. I don't think the original question was so frequently asked, so I think it's fine to get rid of it.
```suggestion
```